src/model/loader.nit

```diff
@@ -58,6 +58,15 @@ redef class AppConfig
 		var html = proc.process(content).write_to_string
 
 		var track = new Track(title_id, title, html)
+
+		var ls = ini["languages"]
+		if ls != null then
```

Comment:
I like the "all languages are accepted" but the frontend need to know the list of all these languages. The API should reflect it.

@privat, once you have decided of a policy, you should open an issue about that.